test/reactjs/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "reactjs",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@craco/craco": "^6.4.3",
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^13.0.0",
+    "@testing-library/user-event": "^13.2.1",
+    "react": "^18.1.0",
+    "react-dom": "^18.1.0",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.0"
```

Comment:
Maybe you should also add
```suggestion
    "web-vitals": "^2.1.0"
    "casbin-core": "file:../../"
```

Reply:
@Zxilly do you know how I can fix `ReferenceError: await is not defined` 😅

Reply:
@Zxilly This code looks great to me.

Reply:
> Maybe you should also add

Using  `"casbin-core": "link:../../"` for refer to casbin-core/lib.